doc/release-procedure.md

```diff
@@ -0,0 +1,21 @@
+# Uno.Core release procedure
+
+Uno.Core uses [GitVersion])(https://gitversion.readthedocs.io/en/latest/) for its versioning, in `mainline` mode for the `release/stable` branches, and **ContinuousDeployment** for the `master` branch.
+
+Tagging is the main driver for planning releases.
+
+## Branches 
+- On the `master` branch, the main development is happening and is considered unstable. The nuget packages produced end with `-dev.X`, where X is the number of commits since the last initiated release.
+- On the `release/beta` branch, stabilisation occurs to get a stable release. The version is inherited from the branch point from master. The nuget packages produced end with `-beta.X`, where X is the number of commits since the last stable release.
+- On the `release/stable` branch, stable nuget packages are produced for each pushed merge or commit. The **Patch** number is increased by the number of commits since the release tagging occured.
+
+## Creating a release
+
+### When planning for a beta
+- Once a release is planned, make a branch in `release/beta` (e.g. `release/beta/1.29`), and tag the commit using the requested version (e.g. `1.29`). Tagging will automatically the version in the `master` increased by a **minor** number.
```

Comment:
"Tagging will automatically the version in the `master` increased by a **minor** number"

Not sure to understand

Reply:
This means that gitversion will automatically detect that a stable version will be produced and a bump the version when run on master.